src/components/Timer/Timer.js

```diff
@@ -21,10 +63,13 @@ const Timer = () => {
         setIsRunning((prev) => !prev);
     };
 
+    const timeToDisplay = convertFromSecondToMinutes(
+        state[intervalOrder[intervalElement]]
```

Comment:
for clarity we may want to separate a few constants here

```suggestion
        const interval = intervalOrder[intervalElement]];
        const timer = state[interval];
        const timeToDisplay = convertFromSecondToMinus(timer);
```